### DISCUSSION TITLE
What will become of OnlyOffice (or what's left of it)?

### DISCUSSION THREAD

**Discussion by k4my4b** [nextcloud/all-in-one#1379] [Questions] [answered]
Simply put, will the remnants of OnlyOffice be kept in AIO containers for (backwards)compatibility/etc. or do you plan to get rid of them?
I only ask because I am running AIO through manual installation mode (solely I can have OO (solely because collabora doesn't have mail merge)) and I have watchtower running to keep everything up to date, and I fear one of these days support will be dropped without me knowing and ...

**Comment by szaimen** [comment, marked as answer]
Hi, first of all: it is really not recommended to use watchtower to update the AIO containers. It is just a matter of time until that will break on you. See https://github.com/nextcloud/all-in-one/tree/main/manual-install#how-to-update for the correct update procedure.

Second, by following the update procedure, you will compare the docker-compose file of the repo with your own one. When we remove the container eventally, it also will get removed from the file of the repo and thus you will see that it got removed.

**Comment by Aquillum** [comment]
Hi, I found and installed AIO about 1 week ago because none of the other nextcloud HowTo's worked for me in combination with OnlyOffice.
So I used the manual install docker-compose file and the stack worked out-of-the-box flawlessly ! And it was automatically configured in nextcloud with the correct entrys in the configuration fields - a nightmare before !
At the end I managed to pull OO magically also with the Mastercontainer by just setting 0->1 in a file and a restart of the containers via AIO :) 

I would love if OO support will not be dropped completely beacause AIO runs perfectly stable with it behind my cloudflare tunnel.
IF it would be dropped I will have to pull the offical container next to the AIO ones and hope to get internal network/ssl/link stuff right which was a nightmare before AIO !

Thank you for your awesome work !
